apps/documents/tasks.py

```diff
@@ -41,10 +98,33 @@ def upload_files_to_vector_store_task(collection_file_ids: list[int], chuking_st
 
```

Comment:
If there was an error it could be worth recording it on the file to show to the user.

I'm thinking there could be two kinds of errors, temporary ones like API down / rate limits etc. and permanent ones like unable to read file or bad file type.

For the temporary ones the user could retry the indexing.

Reply:
I think I'll do this in a separate PR. I'll create an issue for this